auto_rx/autorx/aprs.py

```diff
@@ -653,7 +652,7 @@ def upload_timer(self):
         """ Add packets to the aprs upload queue if it is time for us to upload. """
 
         while self.timer_thread_running:
-            if int(time.time()) % self.synchronous_upload_time == 0:
+            if datetime.datetime.now() > self.next_upload:
```

Comment:
This is not much better, it now delays positions for up to 60 seconds, and they will arrive badly delayed. Other receivers have forwarded the same positions earlier.

A position that is received, must be forwarded _immediately_, and then additional positions from the same station should be suppressed for 30 seconds.

Reply:
60 seconds would be the max delay. It'll be uploading the latest sonde frame at that time. This is used to so we always report the last sonde frame. It's a bit hacky but typically the frame will be less than 1-2 seconds old.